### ISSUE TITLE
Button_Switch_SMD.pretty naming convention not continuous

### ISSUE THREAD

**Issue by pointhi** [KiCad/kicad-footprints#144]
We have some mixture of rotary encoder without a prefix, switches without a ```SW_SPST``` prefix and switches with/without manufactur name

**Comment by Ratfink** [comment]
Also: a lot of pushbutton switches are currently prefixed with `SW_SPST`.  Should they be renamed to use an `SW_Push` prefix?

**Comment by Shackmeister** [comment]
SW_Push doesn't give the electrical configuration of the switch. A pushbutton could also be SPDT, SPDT etc

**Comment by Ratfink** [comment]
Good point.  I think it would be best to put both in the name, which would allow for better footprint filters.  Really, our switch naming needs a lot of improvement and standardization, and it should extend to the symbols as well.

**Comment by poeschlr** [comment]
We can think about a scheme that is used for new contributions. But existing symbols and footprints should not be changed in a manner that would break existing projects that use them. (Otherwise we are unable to supply users with bugfix updates of the library.)

**Comment by poeschlr** [comment]
We have documented a naming convention for new contributions here: https://github.com/KiCad/kicad-symbols/issues/580
